cmd/ketch/app_list.go

```diff
@@ -43,11 +44,19 @@ func appList(ctx context.Context, cfg config, out io.Writer) error {
 		poolsByName[pool.Name] = pool
 	}
 	w := tabwriter.NewWriter(out, 0, 4, 4, ' ', 0)
-	fmt.Fprintln(w, "NAME\tPOOL\tUNITS\tADDRESSES\tDESCRIPTION")
+	fmt.Fprintln(w, "NAME\tPOOL\tSTATE\tADDRESSES\tDESCRIPTION")
 	for _, item := range apps.Items {
+		pods, err := cfg.KubernetesClient().CoreV1().Pods(item.Namespace).List(ctx, metav1.ListOptions{
```

Comment:
let's get pods of all ketch apps in one call and group them by an app.

Reply:
namespaces....

Reply:
https://github.com/datawire/ambassador/blob/master/pkg/k8s/client.go#L36